components/flex/style/index.less

```diff
@@ -0,0 +1,127 @@
+@import '../../style/themes/index';
+@import '../../style/mixins/index';
+
+@flex-prefix-cls: ~'@{ant-prefix}-flex';
+
+.@{flex-prefix-cls} {
+  display: flex;
+  gap: var(--flex-gap, 0);
+  flex: var(--flex, unset);
+
+  &-direction {
+    &-row {
+      flex-direction: row;
+    }
+
+    &-column {
+      flex-direction: column;
+    }
+
+    &-row-reverse {
+      flex-direction: row-reverse;
+    }
+
+    &-column-reverse {
+      flex-direction: column-reverse;
+    }
+  }
```

Comment:
ant-design only supports two directions, horizontal and vertical. We don't need to support so many directions.

Additionally, `nzDirection` needs to be changed to `nzVertical: boolean`